dask_bigquery/core.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import annotations
+
+from contextlib import contextmanager
+from functools import partial
+
+import pandas as pd
+import pyarrow
+from dask.base import tokenize
+from dask.dataframe.core import new_dd_object
+from dask.highlevelgraph import HighLevelGraph
+from dask.layers import DataFrameIOLayer
+from google.cloud import bigquery, bigquery_storage
+
+
+@contextmanager
+def bigquery_client(project_id=None):
```

Comment:
now that this always returns both BQ and BQS, should it be called `bigquery_clients`? 🙂